### PR TITLE
doc_config_config_rule

### PR DESCRIPTION
**Pull request by alex-torres1980**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_config_config_rule/config_config_rule.tf [hunk 1]
```diff
@@ -0,0 +1,37 @@
+resource "awscc_config_config_rule" "r" {
+  config_rule_name = ""
```
Comment (wellsiau-aws) [inline examples/resources/awscc_config_config_rule/config_config_rule.tf:2]: empty string for resource name is not recommended, please add some meaningful resource name

File: examples/resources/awscc_config_config_rule/config_config_rule.tf [hunk 2]
```diff
@@ -0,0 +1,37 @@
+resource "awscc_config_config_rule" "r" {
```
Comment (wellsiau-aws) [inline examples/resources/awscc_config_config_rule/config_config_rule.tf:1]: I would avoid ambiguous resource logical id such as `r` , can you please look at other example, at minimum you can use something like `example`

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
thanks for your contribution, please check couple suggestions above

**Review by wellsiau-aws — APPROVED** [review]
thank you for the contribution, LGTM now

**Comment by marcosentino** [comment]
@alex-torres1980 Thank you for the contribution!

**Pull request merged**